hackathon_site/hardware/models.py

```diff
@@ -102,7 +102,7 @@ class Order(models.Model):
         ("Cancelled", "Cancelled"),
     ]
 
-    hardware_set = models.ManyToManyField(Hardware, through=OrderItem)
+    hardware = models.ManyToManyField(Hardware, through=OrderItem)
```

Comment:
In my last comment I meant to just rename it in the serializer, I didn't realize we named the field `hardware_set` in the model to begin with :thinking:. I like this change (since in most cases we should be specifying an explicit related name rather than the default `<field>_set`).

Since you changed a model though, you will need to make a migration:

```bash
$ python manage.py makemigrations
```

Format the migration that gets created with black, and commit it.